docs/apis/avalanchego/apis/p-chain.md

```diff
@@ -908,23 +916,27 @@ curl -X POST --data '{
 
 ### platform.getCurrentSupply
 
-Returns an upper bound on the number of AVAX that exist. This is an upper bound because it does not account for burnt tokens, including transaction fees.
+Returns an upper bound on amount of tokens that exist that can stake the requested subnet. This is an upper bound because it does not account for burnt tokens, including transaction fees.
```

Comment:
```suggestion
Returns an upper bound on amount of tokens existed that can stake the requested Subnet. This is an upper bound because it does not account for burnt tokens, including transaction fees.
```

Reply:
I don't think "existed" is grammatically correct here.

Reply:
@danlaine can you help here?